api/src/identity-access-management/domain/usecases/authenticate-oidc-user.usecase.js

```diff
@@ -21,10 +21,11 @@ import { ForbiddenAccess } from '../../../shared/domain/errors.js';
 async function authenticateOidcUser({
   audience,
   code,
+  state,
+  iss,
   identityProviderCode,
   nonce,
   sessionState,
```

Comment:
Suggestion : trier les paramètres de fonctions par ordre alphabétique par rapport aux standards 😄 

```suggestion
  audience,
  code,
  identityProviderCode,
  iss,
  nonce,
  sessionState,
  state,
```
A voir pour faire une règle automatisée eslint 🤔

Reply:
Et je dirais que ça vaut le coup de le faire aussi dans `oidc-provider.controller.js` à l'appel du usecase et à la déstructuration du payload pour être cohérent et faciliter la lecture.